classes/session/SessionManager.php

```diff
@@ -267,12 +258,11 @@ public function regenerateSessionId(): bool
         $success = false;
         $currentSessionId = session_id();
 
-        if (session_regenerate_id() && isset($this->userSession)) {
+        if (session_regenerate_id(true) && isset($this->userSession)) {
             // Delete old session and insert new session
             $this->sessionDao->deleteById($currentSessionId);
             $this->userSession->setId(session_id());
             $this->sessionDao->insertObject($this->userSession);
-            $this->updateSessionCookie(); // TODO: this might not be needed on >= 4.3.3
```

Comment:
The `session_regenerate_id()` above issues a `Set-Cookie`, so I guess this isn't needed.

Reply:
On my test the `session_regenerate_id()` was issuing a `Set-Cookie` under the hood, but I'll double check.